server/service/assessment/create.ts

```diff
@@ -28,4 +29,12 @@ export const create = async (
     )
     return createdAssessment
   })
+
+  await SettingsRepository.update({
+    settings: {
+      defaultAssessmentId: ret.id,
+    },
+  })
```

Comment:
this shouldn't happen at assessment creation. it's something we have to trigger programmatically